core/src/edu/cornell/gdiac/temporary/Level.java

```diff
@@ -0,0 +1,284 @@
+package edu.cornell.gdiac.temporary;
+
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.math.Vector2;
+import com.badlogic.gdx.utils.JsonValue;
+import com.badlogic.gdx.utils.Queue;
+import edu.cornell.gdiac.assets.AssetDirectory;
+import edu.cornell.gdiac.audio.MusicQueue;
+import edu.cornell.gdiac.temporary.entity.BandMember;
+import edu.cornell.gdiac.temporary.entity.Note;
+
+public class Level {
+    public BandMember[] getBandMembers() {
+        return BandMembers;
+    }
+
+    public Texture getHitNoteTexture() {
+        return hitNoteTexture;
+    }
+
+    public Texture getSwitchNoteTexture() {
+        return switchNoteTexture;
+    }
+
+    public Texture getHoldNoteTexture() {
+        return holdNoteTexture;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public int getOrder() {
+        return order;
+    }
+
+    public int getMaxCompetency() {
+        return maxCompetency;
+    }
+
+    public int getSpawnOffset() {
+        return spawnOffset;
+    }
+
+    public MusicQueue getMusic() {
+        return music;
+    }
+
+    public void setBandMembers(BandMember[] bandMembers) {
+        BandMembers = bandMembers;
+    }
+
+    public void setHitNoteTexture(Texture hitNoteTexture) {
+        this.hitNoteTexture = hitNoteTexture;
+    }
+
+    public void setSwitchNoteTexture(Texture switchNoteTexture) {
+        this.switchNoteTexture = switchNoteTexture;
+    }
+
+    public void setHoldNoteTexture(Texture holdNoteTexture) {
+        this.holdNoteTexture = holdNoteTexture;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public void setOrder(int order) {
+        this.order = order;
+    }
+
+
+    public void setMaxCompetency(int maxCompetency) {
+        this.maxCompetency = maxCompetency;
+    }
+
+    public void setSpawnOffset(int spawnOffset) {
+        this.spawnOffset = spawnOffset;
+    }
+
+    public void setMusic(MusicQueue music) {
+        this.music = music;
+    }
+
+    private BandMember[] BandMembers;
+    private Texture hitNoteTexture;
+    private Texture switchNoteTexture;
+    private Texture holdNoteTexture;
+    private Texture holdEndTexture;
+    private Texture holdTrailTexture;
+
+    private Texture hpbar;
+    private String title;
+    private int order;
+    private int maxCompetency;
+    private int spawnOffset;
+   private MusicQueue music;
+
+    /**
+     * The last sample that health was decremented due to continuous decay
+     */
+    private long lastDec;
+    public Level(JsonValue data, AssetDirectory directory) {
+        //Read in Json  Value and populate asset textures
+        lastDec = 0;
+        title = data.getString("title");
```

Comment:
wait did you change the JSON format? I am the one who was supposed to be deciding the JSON format you can't just change it. My level editor saves the level into the format I specified. Most of the differences are names of the fields, but for held notes I don't have a "connections" field, I have a "sample" field which I call "position" and "duration". You gotta change that before we merge.